GoogleForCreators/web-stories-wp
# Fixed thumbnail sizes

## Context

Fixed thumbnail sizes according to designs.

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2021-02-12 at 00 55 50](https://user-images.githubusercontent.com/637548/107735174-1ae14b00-6ccd-11eb-853f-8a6c1a98c08b.png) | ![Screen Shot 2021-02-12 at 00 55 35](https://user-images.githubusercontent.com/637548/107735176-1c127800-6ccd-11eb-9a23-52b8a7bffcc6.png) |

## Testing Instructions



### QA

This PR can be tested by following these steps:

1. Add a perfect circle to a page
2. Observe that the circle is still perfectly round in the thumbnail regardless of thumbnail size (the thumbnail is larger for large windows).

### UAT




## Checklist




---

Fixes #6358
